application/console/migrations/m160414_140140_email_queue.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use yii\db\Schema;
+use yii\db\Migration;
+
+class m160414_140140_email_queue extends Migration
+{
+    // Use safeUp/safeDown to run migration code within a transaction
+    public function safeUp()
+    {
+        $this->createTable(
+            '{{email_queue}}',
+            [
+                'id' => 'pk',
+                'to_address' => 'varchar(255) not null',
+                'cc_address' => 'varchar(255) null',
+                'subject' => 'varchar(255) not null',
+                'text_body' => 'text null',
+                'html_body' => 'text null',
+                'attempts_count' => 'tinyint not null default 0',
```

Comment:
Is there any noticeable risk that someone trying to hack the system could get the `attempts_count` to overflow the range of a tinyint? I just know that's one of the examples used in textbooks about how an error counter can exceed it's data type size and get back to a state making it seem like there were no errors (or in our case, no attempts). Probably a pretty unlikely situation, but the use of tinyint (0 - 255) brought that to mind.